temoa/extensions/get_comm_tech.py

```
@@ -297,7 +297,7 @@ def get_info(inputs):
     file_ty = re.search(r'(\w+)\.(\w+)\b', inp_file)  # Extract the input filename and extension
 
     if not file_ty:
-        raise Exception('The file type {} is not recognized.'.format(file_ty))
+        raise Exception(f'The file type {file_ty} is not recognized.')
```

Ruff (0.14.3)

Create your own exception

(TRY002)